index.js

```diff
@@ -0,0 +1,36 @@
+const jokeCards = document.getElementById("jokeCards");
+const reloader = document.getElementById("reload");
+
+const loadJokes = async () => {
```

Comment:
Do you mind rewriting this solution with promises rather than async/await? We don't intend to show solution code to the learners but it could be a useful reference for future instructors, and it is also a good reality check for us to double confirm that the lab can be completed with the concepts we have taught them.